### PR TITLE
Use serialized_all for the requested node on the story API endpoint

### PR DESCRIPTION
**Pull request by yashvardhannanavati**

### CODE REVIEW

**Comment by yashvardhannanavati** [comment]
The code works but I think we will have to fix the tests

File: purview/utils/general.py [hunk 1]
```diff
@@ -178,7 +178,7 @@ def create_query(item, uid_name, uid, reverse=False):
     return query
 
 
-def query_neo4j(query):
+def query_neo4j(query, resource):
```
Comment (mprahl) [inline purview/utils/general.py]: This function was written to be pretty generic and by adding this parameter it makes it specific to what the story API needs. How about making this `resources_to_expand` and have it be an optional list of labels the developer can provide that will have `serialized_all` be called on?

Comment (yashvardhannanavati) [reply purview/utils/general.py]: hm, how about I make an optional param which will have to be a list, and if it's present then '''python query_neo4j ''' will expand that node, else no.

Comment (yashvardhannanavati) [reply purview/utils/general.py]: Oh, I think we are telling the same thing. Sorry!

File: purview/utils/general.py [hunk 2]
```diff
@@ -178,7 +178,7 @@ def create_query(item, uid_name, uid, reverse=False):
     return query
 
 
-def query_neo4j(query):
+def query_neo4j(query, resources_to_expand=None):
```
Comment (mprahl) [inline purview/utils/general.py]: At the beginning of the function, you should add:
```python
if resources_to_expand is None:
    resources_to_expand = []
```

**Comment by mprahl** [comment]
:+1:

**Pull request merged**